src/assets/TokenDetectionController.ts

```diff
@@ -1,12 +1,18 @@
+// eslint-disable-next-line import/no-named-as-default
+import AbortController from 'abort-controller';
 import { BaseController, BaseConfig, BaseState } from '../BaseController';
 import type { NetworkState, NetworkType } from '../network/NetworkController';
 import type { PreferencesState } from '../user/PreferencesController';
-import { safelyExecute, toChecksumHexAddress } from '../util';
+import {
+  safelyExecute,
+  toChecksumHexAddress,
+  isTokenDetectionEnabledForNetwork,
+} from '../util';
 import { MAINNET } from '../constants';
-
+import { NetworksChainId } from '..';
 import type { TokensController, TokensState } from './TokensController';
 import type { AssetsContractController } from './AssetsContractController';
-import { Token } from './TokenRatesController';
+import { Token } from './TokenListController';
```

Comment:
TokenListController is exporting `RawToken` instead of `Token`

Reply:
I reverted this change. Not needed for now.